src/components/Layout/Layout.tsx

```diff
@@ -318,7 +325,10 @@ export const Layout = ({
               </View>
               <View key={asPathWithNoHash} className="layout-main">
                 <Flex
+                  id={mainId}
                   as="main"
+                  tabIndex={-1}
+                  aria-label="Main content"
```

Comment:
Adding an aria-label so it doesn't label it with the first element it finds and instead announces "Main content".